src/jobs.py

```diff
@@ -294,7 +297,7 @@ async def send_reminder_now(context: CallbackContext) -> None:
     consultation = job_data.consultation
     telegram_id = consultation.telegram_id
     consultation_count = await service.get_consultations_count(telegram_id)
-    text = get_reminder_text(job_data, *consultation_count)
+    text = get_reminder_text(job_data, consultation_count)
```

Comment:
```suggestion
    text = get_reminder_text(job_data, **consultation_count)
```
Благодаря изменениям в функции get_reminder_text и get_consultations_count мы сможем просто разворачивать словарь при вызове функции